.github/workflows/release.yml

```diff
@@ -56,6 +82,14 @@ jobs:
           use-mamba: true
           python-version: "3.8"
           channel-priority: strict
+      - name: Update platform tag for aarch64 wheel
```

Comment:
If this is working around a bug or other issue, it would be good to leave some comments or a link to an issue describing that.

Reply:
Agreed, will leave a comment linking to the glibc bug itself:

https://bugzilla.redhat.com/show_bug.cgi?id=1722181

Reply:
Looks like there is now a comment above this line with that note. Marking this resolved